python/afdko/buildcff2vf.py

```diff
@@ -156,14 +152,12 @@ def make_curve_coords(self, coords, is_default):
         if is_default:
```

Comment:
You're not using `prev_coords` below, so you don't need `prev_cmd` in line 151 above.